sstables/sstables_manager.hh

```diff
@@ -102,6 +102,7 @@ public:
         utils::updateable_value<uint32_t> column_index_auto_scale_threshold_in_kb = utils::updateable_value<uint32_t>(10240);
         utils::updateable_value<double> memory_reclaim_threshold = utils::updateable_value<double>(0.2);
         const std::vector<sstring>& data_file_directories;
+        utils::updateable_value<sstring> format = utils::updateable_value<sstring>(fmt::to_string(sstable_version_types::me));
```

Comment:
We should add a constant to `sstable/version.hh` with default version for writes, so when we bump the default, it is updated both in db config and here. As it-is, https://github.com/scylladb/scylladb/pull/26377 will already make this out-of-date.
Mostly a cosmetic issue, since all relevant users overwrite with value from db config, so this can be a follow-up.